app/helpers/badge_helper.rb

```diff
@@ -0,0 +1,21 @@
+module BadgeHelper
+  def badge_colour(offender)
+    if offender.indeterminate_sentence?
+      'purple'
+    else
+      'blue'
+    end
+  end
+
+  def badge_text(offender)
+    if offender.indeterminate_sentence?
+      'Indeterminate'
+    else
+      'Determinate'
+    end
+  end
+
+  def badge_for_parole?(offender)
+    offender.tariff_date || offender.parole_eligibility_date
```

Comment:
kt's more idiomayic to use present? for this  rathet than treat dates as booleans